tests/rptest/tests/cluster_quota_test.py

```diff
@@ -176,7 +176,7 @@ def produce(self, producer, amount):
         for f in response_futures:
             f.get(1)
 
-    def fetch(self, consumer, messages_amount, timeout_sec=300):
+    def fetch(self, consumer, messages_amount, timeout_sec=15):
```

Comment:
Why does this value change?

Reply:
I changed it while testing to timeout faster on failure. Locally 15s was plenty, but I'll change it back to 300 just to avoid any CI flakiness.